---
layout: page
title: About
---

Liam Childs' Homepage
=====================

Research Interests
------------------

### Next-Generation sequencing analysis

Currently I'm working on the OncoPath project. Current goals include the annotation of sequence variants detected in colo-rectal cancer samples.

### Genome Wide Association Studies

I'm interested in developing tools that help integrate and disseminate data produced by GWAS. Recently I developed and produced Matapax, an online pipeline to make GWAS available to all Arabidopsis researchers. I am currently working on making several improvement to Matapax to improve run-time, ease-of-use and hope to eventually develop a database to store results that Matapax produces.
RNA regulation of gene expression

Investigating several ways in which RNA structure may regulate the transcription and translation of genes in general, including phenomenon such as riboswitches and structured UTR regions in prokaryotic and eukaryotic genomes.

### Prediction of ncRNA genes

Investigated the use of graph properties based on RNA structure in the prediction of ncRNA genes and their functions. The investigations culminated in the implementation of a tool called GraPPLE that provides predictions on the functionality of RNA sequences such as whether the molecule is functional or not and if so, then a possible function that it could take. This prediction is accompanied with a confidence measure.
Effect of Polymorphisms on RNA Structure and Genes

By using tiling arrays to study several different Arabidopsis thaliana accessions we are able to predict Single Feature Polymorphisms (SFPs). Upon mapping these SFPs to the Arabidopsis genome we can investigate the effects that they have on potentially regulatory RNA structure.

Academic History
----------------
2006 - 2009\\
PhD (Bioinformatics)\\
"Bioinformatics approach to analysing RNA mediated regulation of gene expression"\\
University of Potsdam ([[http://www.uni-potsdam.de/english/|English]],[[http://www.uni-potsdam.de/|Deutsch]])\\
Potsdam\\
Germany\\

2000 - 2004\\
B.Sc. (Bioinformatics) Honours 1st class\\
Sydney University\\
Sydney\\
Australia

Employment History
------------------

2013 - present\\
Post Doctoral Fellow\\
Humboldt University\\
Berlin\\
Germany

2009 - 2013\\
Post Doctoral Fellow\\
Max-Planck Institute for Molecular Plant Physiology\\
Potsdam\\
Germany

2004 - 2006\\
Bioinformatics Data Analyst\\
CSIRO Plant Industry\\
Black Mountain\\
Australia

Tools and Skills
----------------

### Programming

Languages: C, C++, C#, Java, Python, SQL, R, Silverlight, Javascript

Paradigms: Waterfall, Test Driven Development, Agile Programming

Web technologies: web2py, cgi, ajax, rest

### Association Mapping

TASSEL, GAPIT, EMMA

### Tiling Arrays

TAS
General CEL file analysis using R (Bioconductor) and Python.

Contact
-------

### Address

Knowledge Management in Bioinformatics\\
Institute for Computer Science\\
Humboldt-Universitaet zu Berlin\\
Rudower Chaussee 25\\
12489 Berlin

### Email

liam (dot) childs (at) hu (minus) berlin (dot) de

Publications
------------

Zhang J, Ruf S, Hasse C, Childs LH, Scharff LB and Bock R (2012) Identification of cis-elements conferring high levels of gene expression in non-green plastids,The Plant Journal.

Zuther E, Schulz E, Childs LH, Hincha DK (2012). Clinal variation in the nonacclimated and cold acclimated freezing tolerance of Arabidopsis thaliana accessions. Plant, cell & environment.

Childs LH, Lisec J, Walther D (2012). Matapax: An Online High-Throughput Genome-Wide Association Study Pipeline. Plant Phys. 158: 1-8. doi: ​10.​1104/​pp.​112.​194027

Ryngajllo M, Childs LH, Lohse M, Giorgi FM, Lude A, Selbig J, Usadel B (2011) SLocX: Predicting subcellular localization of Arabidopsis proteins leveraging gene expression data. Frontiers in Plant Science

Childs LH, Scharff LB , Walther D, Bock R, (2011) Local Absence of Secondary Structure Permits Translation of mRNAs that Lack Ribosome-Binding Sites. PLoS Genet 7(6): e1002155. doi:10.1371/journal.pgen.1002155

Lohse M, Nunes-Nesi A, Krueger P, Nagel A, Hannemann J, Giorgi FM, Childs LH, Osorio S, Walther D, Selbig J, Sreenivasulu N, Stitt M, Fernie AR, Usadel B (2010) Robin: An intuitive wizard application for R-based expression microarray quality assessment and analysis. Plant Physiology

Childs LH, Witucka-Wall H, Günther T, Sulpice R, von Korff M, Stitt M, Walther D, Schmid KJ, and Altmann T (2010) Single feature polymorphism (SFP)-based selective sweep identification and association mapping of growth-related metabolic traits in Arabidopsis thaliana. BMC Genomics 11: 188

Childs LH, Nikoloski Z, May P, Walther D (2009) Identification and classification of ncRNA molecules using graph properties. Nucl. Acids Res. 37(9): e66. doi:10.1093/nar/gkp20

Software
--------

[GraPPLE][grapple] (originally: The RNA sorting hat)

[Matapax][matapax]

[SoFIA][sofia]

GitHub Projects
---------------

[childsish@github](http://github.com/childsish)

[SoFIA](https://githib.com/childsish/sofia)

[Dynamic PyYAML](http://github.com/childsish/dynamic-pyyaml)


[grapple]: http://grapple.mpimp-golm.mpg.de
[matapax]: http://matapax.mpimp-golm.mpg.de
[sofia]: https://github.com/childsish/sofia
